ci-operator/config/open-cluster-management/prometheus/open-cluster-management-prometheus-release-2.4.yaml

```diff
@@ -0,0 +1,69 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.15-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.15-linux
+  open-cluster-management_builder_go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
```

Comment:
Although this image is referenced by the Dockerfile.prow for the prometheus image, it's not needed as you specify the full builder image in the docker file.